base/stats.go

```diff
@@ -1909,6 +1911,10 @@ func (d *DbStats) InitSharedBucketImportStats() error {
 		if err != nil {
 			return err
 		}
+		resUtil.ImportProcessCompute, err = NewFloatStat(SubsystemSharedBucketImport, "import_process_compute", labelKeys, labelVals, prometheus.GaugeValue, 0)
+		if err != nil {
+			return err
+		}
```

Comment:
I wonder if these types of stats should actually be in a serverless section that is only enabled/calculated when the serverless flag is enabled.

This has no benefit/makes no sense for an on-prem deployment.